frontend/src/components/pages/Scheduling/types.ts

```diff
@@ -74,4 +90,4 @@ export const timeRanges = {
   night: ["9:00 PM - 10:00 PM", "10:00 PM - 11:00 PM", "11:00 PM - 12:00 AM"],
 };
 
-export const frequencies = ["One time", "Daily", "Weekly", "Monthly"];
+export const frequencies = ["One time donation", "Daily", "Weekly", "Monthly"];
```

Comment:
is there a reason why frequencies is an array here and not an enum? i noticed we already had an enum and i think it might be clearer since i can't really tell what ```frequencies[0]``` or ```frequencies[1]``` is clearly

Reply:
not sure, @AmandaMisjuwar might be able to speak to it better but I think it might be getting mapped somewhere which is why it's an array?